.github/workflows/release-azure-pipelines.yml

```diff
@@ -93,40 +93,268 @@ extends:
               targetPath: dist/${{ config.runtime }}
               artifactName: azureauth-${{ parameters.version }}-${{ config.runtime }}
           steps:
-            - checkout: self
-            - task: UseDotNet@2
-              displayName: Use .NET Core sdk 6.x
-              inputs:
-                version: 6.x
+          - checkout: self
+          - task: UseDotNet@2
+            displayName: Use .NET Core sdk 6.x
+            inputs:
+              version: 6.x
+          - task: NuGetToolInstaller@0
+            displayName: Use NuGet 6.x
+            inputs:
+              versionSpec: 6.x
+          - task: DotNetCoreCLI@2
+            displayName: Install dependencies
+            inputs:
+              command: restore
+              feedsToUse: select
+              vstsFeed: $(vstsFeedId)
+              includeNuGetOrg: false
+              arguments: --runtime ${{ config.runtime }}
+          - task: DotNetCoreCLI@2
+            displayName: Test
+            inputs:
+              command: test
+              arguments: --configuration release --no-restore
+          
+          - task: DotNetCoreCLI@2
+            displayName: Build artifacts
+            env:
+              ADO_TOKEN: $(System.AccessToken)
+            inputs:
+              command: publish
+              projects: src/AzureAuth/AzureAuth.csproj
+              arguments: -p:Version=${{ parameters.version }} --configuration release --self-contained true --runtime ${{ config.runtime }} --output dist/${{ config.runtime }}
+              publishWebProjects: false
+              zipAfterPublish: false
+              modifyOutputPath: true

```

Comment:
Sorry if this is a dumb question - Is the [Analyze](https://github.com/AzureAD/microsoft-authentication-cli/blob/ea5bf2cfd9db97e96394f5051a033aac4d32b23c/.github/workflows/release.yml#L88C3-L122C37) job which does CodeQL analysis already included in the 1ES pipeline template?

Reply:
That's a good question, Manuha. Yes, SDL Analysis is auto injected by 1ES PT.